test/game-test.js

```diff
@@ -0,0 +1,50 @@
+import chai from 'chai';
+const expect = chai.expect;
+import Puzzle from '../src/puzzle.js'
+import Turn from '../src/turn.js'
+import testData from '../data/test_data_set.js'
+import Round from '../src/round.js';
+import Player from '../src/player.js';
+import Wheel from '../src/wheel.js'
+import Game from '../src/game.js'
+
+
+describe('Game', function() {
+  let player1;
+  let player2;
+  let player3;
+  let game;
+  beforeEach(function() {
+    player1 = new Player('Him', 1)
+    player2 = new Player('Her', 2)
+    player3 = new Player('Mantis', 3)
+    game = new Game(player1, player2, player3)
+  });
+
+  it('be an instance of round', function() {
+    expect(game).to.be.an.instanceOf(Game)
+  });
+
+  it('it select puzzles', function() {
+    expect(game.selectedPuzzles.length).to.equal(0)
+    game.makeSelectedPuzzle(testData)
+    expect(game.selectedPuzzles.length).to.equal(4)
+  });
+
+  it('it select puzzles', function() {
```

Comment:
Let's update this description to reflect that we are expecting in our assertions, which is that we are getting 4 puzzles back - 1 with 1 word, 1 with 2 words, etc. (So a summarized version)